### DISCUSSION TITLE
How do I update my filtered repo with the new status of the original repo?

### DISCUSSION THREAD

**Discussion by aabuderi** [newren/git-filter-repo#349] [Q&A] [answered]
I have been using filter-repo in order to share a part of my repository with a client, and I would like to be able to update the sub-repo with changes from the original repo as time goes on. How do I do this? Thanks!

**Comment by newren** [comment, marked as answer]
See issue #66, issue #93, issue #293, issue #329, and https://github.com/josh-project/josh.

**Reply by newren** [reply]
...and https://github.com/newren/git-filter-repo/blob/main/Documentation/FAQ.md#Bidirectional-development-between-a-filtered-and-unfiltered-repository
